src/pudl/transform/ferc1.py

```diff
@@ -2893,18 +2893,22 @@ def process_xbrl_metadata(
         naming conventions...). We use the same rename dictionary, but as an argument to
         :meth:`pd.Series.replace` instead of :meth:`pd.DataFrame.rename`.
         """
-        tbl_meta = super().process_xbrl_metadata(
-            xbrl_metadata_converted, xbrl_calculations
+        tbl_meta = (
+            super()
+            .process_xbrl_metadata(xbrl_metadata_converted, xbrl_calculations)
+            .assign(utility_type="electric", plant_status="in_service")
```

Comment:
Maybe I am remembering incorrectly, but weren't there 2 oddball facts in this table for which `plant_status` wasn't actually `in_service`?

Reply:
Yes exactly, see below on lines 2907 and 2911 where they're getting manually set to `pd.NA` (possibly the source of the mixed NA issue...)

Reply:
Ah right whoops.